feat: update nextjs icon

chrisgrabinski opened the pull request:
The Next.js logo has recently been redesigned and [unveiled shortly before Next.js Conf 2022](https://vercel.com/blog/nextjs-conf-2022).

This updated icon is based on the `apple-touch-icon.png` from https://nextjs.org

| Original | Vector recreation | 
|---|---|
|![apple-touch-icon from nextjs.org](https://user-images.githubusercontent.com/958819/200127741-7a0d0a3e-3b19-4dd1-82cc-fa5133e43721.png)|![Vector recreation of apple-touch-icon from nextjs.org](https://user-images.githubusercontent.com/958819/200127761-549dc2ec-da02-48de-b679-6896b5751585.svg)|

The new icon is using a scaled version of the vector recreation:

| Before | After | 
|---|---|
|![Original icon for Next.js](https://user-images.githubusercontent.com/958819/200127781-8d550c9b-50fb-475b-b43c-918e909a67e1.svg)|![Updated icon for Next.js](https://user-images.githubusercontent.com/958819/200127794-62733101-ac83-49e3-b594-a154f0b790b2.svg)|

miguelsolorio reviewed (changes requested):
Thanks for the PR! We try to not use the plain white color as this won't look great on light themes. Mind using the same color as the previous icon?

chrisgrabinski commented:
> Thanks for the PR! We try to not use the plain white color as this won't look great on light themes. Mind using the same color as the previous icon?

Absolutely! I have changed `#fff` to `#64748B` as in the previous icon and I also made some minor tweaks to the gradient for better contrast when used in this small size.

| Before | Adjusted color | Adjusted gradient 
|---|---|---|
|![Updated icon for Next.js](https://user-images.githubusercontent.com/958819/200127794-62733101-ac83-49e3-b594-a154f0b790b2.svg)|![Updated icon for Next.js with previous color](https://user-images.githubusercontent.com/958819/200515295-e2610ef9-4a9b-4cca-9e58-5524dbf21751.svg)|![Updated icon for Next.js with previous color and adjusted gradient](https://user-images.githubusercontent.com/958819/200516468-5285f37c-3901-4b4a-a481-cb7ff7ee244c.svg)|

miguelsolorio reviewed (approved):
Looks great, thanks for the PR!

<img width="197" alt="CleanShot 2022-11-13 at 11 13 53@2x" src="https://user-images.githubusercontent.com/35271042/201539831-55123ad6-275d-4475-a590-0c11282dbd2c.png">
